src/style/dist/themes/default/antd.css

```diff
@@ -25052,3 +25052,6 @@ h3.ant-typography {
 .ant-collapse-header {
   font-weight: 600;
 }
+.ant-list-item-meta-title > a {
+  color: #a6278f;
```

Comment:
is it a new color ? not in existing colors ?

Reply:
⚠️ should be in the color theme.

Reply:
This is the final css file. There is no use for using variables here.

Reply:
oh yes you're right sorry about this